### PR TITLE
Updated docs.js for bug 1237575

### PR DESCRIPTION
**Pull request by yash-iiith**
Added a new method in documenter function 'getDocumentationUrl' which returns the link to the documentation.

### CODE REVIEW

File: src/docs.js [hunk 1]
```diff
@@ -138,6 +138,11 @@ async function documenter(options) {
       });
     });
 
+  function getDocumentationUrl() {
+    let referenceUrl = "https://docs.taskcluster.net/reference/";
```
Comment (imbstack) [inline src/docs.js]: Can we set `referenceUrl` as an option with a default [here](https://github.com/taskcluster/taskcluster-lib-docs/blob/master/src/docs.js#L15)? That way we can override this if we want to at a later point.

I believe the reason the tests are currently failing is due to the use of double quotes here! Switching to single quotes should clear it up :)

**Review by imbstack — CHANGES_REQUESTED** [review]
Thanks for the help!

**Review by imbstack — APPROVED** [review]
Looks great, thanks!

**Comment by yash-iiith** [comment]
Thank you ! Please have a look at PRs in tc-lib-app and tc-hooks also.

**Comment by yash-iiith** [comment]
@imbstack  Just out of curiosity , all the checks did not pass but you approved it. How to get the code to pass those checks?

**Comment by imbstack** [comment]
Travis-ci hides secret environment variables from PRs into repositories from outside repositories. The tests for this package require some secrets to run so they failed for you. I cloned your code locally and ran the tests here before I approved!

(Good question)

**Comment by yash-iiith** [comment]
Oh okay !! Thanks :D

**Pull request merged**